code/game/objects/items/weapons/manuals.dm

```diff
@@ -499,6 +499,165 @@
+
+
+				<h2>Chapters</h2>
+
+				<ol>
+					<li><a href="#Syndicate">The Syndicate</a></li>
+					<li><a href="#Vampire">The Vampire Lords</a></li>
+					<li><a href="#Changeling">The Changeling Hivemind</a></li>
+					<li><a href="#Wizard">The Wizard Federation</a></li>
+					<li><a href="#Cult">The Cult of Nar-Sie</a></li>
+					<li><a href="#Revolution">The Revolutionaries</a></li>
+					<li><a href="#Blob">The Blob Conglomerate</a></li>
+					<li><a href="#Malf">The Free Silicon Hivemind</a></li>
+					<li><a href="#Vox">The Vox Shoal</a></li>
+					<li><a href="#Ninja">The Spider Clan</a></li>
+					<li><a href="#Minor">other minor factions and threats of note</a>
+                    <ol>
+					<li><a href="#Alien">The Alien Hivemind</a></li>
+					<li><a href="#Catbeast">Feral Catbeasts</a></li>
+					<li><a href="#Time">The Time Agency</a></li>
+					<li><a href="#Rambler">Soul Ramblers</a></li>
+					<li><a href="#Plague">Plague Mice</a></li>
+					<li><a href="#Grue">Grues</a></li>
+					<li><a href="#Pulse">Pulse Demons</a></li>
+					<li><a href="#Spider">Giant Space Spiders</a></li>
+                    </ol>
+                    </li>
+				</ol>
+
+
+				<h2><img src='data:image/png;base64,[icon2base64(icon('icons/logos.dmi', "synd-logo"))]' style='position: relative; top: 8;'><a id="Syndicate">The Syndicate</h2>
+
+                <p>The Syndicate are a collaboration of enemies to Nanotrasen. They are the most notorious threat to our space stations as they have made it their official goal to challenge Nanotrasen's monopoly on the sector. They are known to commit every possible crime from thievery, murder, sabotage, and the outright destruction of stations via nuclear devices. Have no qualms about killing them, as they will not hesitate to kill you unless it directly hinders their mission. They are known to work in red, due to the reuse of old soviet spacesuits from the fourth cold war era.
+
+				<h3>Traitors</h3>
+				<p>One of the scariest traits of Syndicate agents is that most of them have infiltrated many of our space stations. Some manage to get hired to work at them, others get conscripted once there. They come equipped with an uplink filled with "telecrystals", or TC for short, usually hidden in their PDA, that lets them transport highly dangerous items aboard the station, usually in the goal of causing sabotage and destruction. Look out for illegal items in their possessions.
```

Comment:
```suggestion
				<p>Being unable to defeat NT in the open, Syndicate agents have opted to infiltrate many of our space stations and now live among us. The Syndicate's vast network of sleeper cells defies all investigation, and as such pre-emptive attempts at discovering infiltrators is discouraged by company policy (see Space Law and Standard Operating Procedure). Agents come equipped with an uplink filled with "telecrystals" (TCs), clearly hidden in their PDAs. With such device, they are able transport highly dangerous items aboard the station, in their foolish attempts to cause wanton sabotage and destruction. Be aware of Syndicate contraband when apprehending suspects.
```

Less valid-hungry, reference to Space Law.

Reply:
The Syndicate is plenty capable of defeating NT in the open, it's just not their preferred tactic (as Nanotrasen probably has more resources).